sdk/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/model/response/TransactionInfoOrderStatus.java

```diff
@@ -0,0 +1,61 @@
+package nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.response;
+
+import kong.unirest.json.JSONObject;
+
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.Money;
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.enums.TransactionStatus;
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.enums.TransactionType;
+
+import java.util.Optional;
+
+/**
+ * this class contains details about a transaction related to order status.
+ */
+public class TransactionInfoOrderStatus {
+    private final String id;
+    private final String paymentBrand;
+    private final TransactionType type;
+    private final TransactionStatus status;
+    private final Money amount;
+    private final String createdAt;
+    private final String lastUpdatedAt;
+
+    public TransactionInfoOrderStatus(JSONObject jsonObject) {
+        this.id = jsonObject.getString("id");
+        this.paymentBrand = jsonObject.getString("paymentBrand");
+        this.type = jsonObject.getEnum(TransactionType.class, "type");
```

Comment:
we should also include 'REFUND' in TransactionType enum since in OrderStatus response we are also supporting 'REFUND' transacations.

Reply:
Added REFUND and CAPTURE.